includes/express-checkout/class-wc-payments-express-checkout-button-handler.php

```diff
@@ -95,6 +95,7 @@ public function init() {
 		add_action( 'before_woocommerce_pay_form', [ $this, 'display_pay_for_order_page_html' ], 1 );
 		add_filter( 'woocommerce_gateway_title', [ $this, 'filter_gateway_title' ], 10, 2 );
 		add_action( 'woocommerce_checkout_order_processed', [ $this->express_checkout_helper, 'add_order_payment_method_title' ], 10, 2 );
+		add_filter( 'woocommerce_get_country_locale', [ $this->express_checkout_helper, 'modify_country_locale' ], 20 );
```

Comment:
There is a `includes/express-checkout/class-wc-payments-express-checkout-ajax-handler.php`, which is used to handle most (if not all?) of the incoming AJAX requests for the express checkout buttons. I think `WC_Payments_Express_Checkout_Ajax_Handler` would be a good spot for this filter, since it's used on AJAX requests.

What do you think about using that class for this filter?